src/Tizen.NUI/src/internal/Transition/TransitionItem.cs

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright(c) 2021 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+namespace Tizen.NUI
+{
+    using System;
+    using System.ComponentModel;
+
+    using Tizen.NUI.BaseComponents;
+
+    /// <summary>
+    /// TransitionItem is an object to set Transition of View pair those have same TransitionTag.
+    /// TransitionItem object is required for each View pair and this is added to the TransitionSet to play.
+    /// </summary>
+    internal class TransitionItem : TransitionItemBase
+    {
+        /// <summary>
+        /// Creates an initialized transition.<br />
+        /// </summary>
+        /// <remarks>DurationmSeconds must be greater than zero.</remarks>
+        public TransitionItem(View source, View destination, TimePeriod timePeriod, AlphaFunction alphaFunction) : this(Interop.TransitionItem.New(source.SwigCPtr, destination.SwigCPtr, timePeriod.SwigCPtr), true)
+        {
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+            AlphaFunction = alphaFunction;
+        }
+
+        internal TransitionItem(global::System.IntPtr cPtr, bool cMemoryOwn) : base(cPtr, cMemoryOwn)
+        {
+        }
+
+        /// <summary>
+        /// Downcasts a handle to transition handle.<br />
+        /// If handle points to an transition object, the downcast produces a valid handle.<br />
+        /// If not, the returned handle is left uninitialized.<br />
+        /// </summary>
+        /// <param name="handle">Handle to an object.</param>
+        /// <returns>Handle to an transition object or an uninitialized handle.</returns>
+        /// <exception cref="ArgumentNullException"> Thrown when handle is null. </exception>
+        public new static TransitionItem DownCast(BaseHandle handle)
```

Comment:
Please remove unused DownCast methods all.

Reply:
Thanks I removed it